src/main/kotlin/com/improve_future/harmonica/core/Connection.kt

```diff
@@ -32,6 +32,9 @@ open class Connection(
                 if (PluginConfig.hasExposed())
                     Database.connect({ coreConnection })
                 else null
+        if (config.dbms == Dbms.SQLite) {
```

Comment:
Can't this code be placed around line 28?
And I think you have to check `PluginConfig.hasExposed()`, for users who don't use exposed.

https://github.com/KenjiOhtsuka/harmonica/issues/76

Reply:
Thank you for your reply!
Changing property of `TransactionManager.manager` before connecting will cause an error so, it will be needed to place after line 33.
Anyway, accessing `TransactionManager` without Exposed will cause `ClassNotFoundException` so it will be needed to check if using Exposed or not as you told.
I'll fix this problem and make a new PR.